### PR TITLE
Add checkbox visible write ins

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?

This integrates Design System's support for visible write-ins in checkbox and mutually exclusive checkbox answers. Checkboxes already share common config with radios in runner so no significant changes were needed. Two unit tests(using mock), functional test and test schema were added to cover the change. Validator supports "visible" option in answer text field definitions for checkbox, but also number and any text type of answers.

### How to review 

Run unit tests and functional test(`./node_modules/.bin/wdio tests/functional/wdio.conf.js --spec tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js`). Check new test schema. Add "visible" option to checkbox text field answer in census schemas.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const MutuallyExclusivePage = require('../../../generated_pages/checkbox_detail_answer_visible/mutually-exclusive.page.js');
+
+
+describe('Given I start a checkbox survey with a write-in option', function() {
+  beforeEach(function() {
+    browser.openQuestionnaire('test_checkbox_detail_answer_visible.json');
+  });
+
+  it('When I view a write-in checkbox and the visible option is set to true, Then the detail answer label should be displayed', function() {
+    expect($(CheckboxVisibleTruePage.otherDetail()).isDisplayed()).to.equal(true);
```
Comment (MebinAbraham) [inline tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js:13]: These should use `.to.be.true;` or `.to.be.false;`

Comment (petechd) [reply tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js]: Done! Thanks👍

File: tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const CheckboxVisibleTruePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-true.page.js');
+const CheckboxVisibleFalsePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-false.page.js');
+const CheckboxVisibleNonePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-none.page.js');
+const MutuallyExclusivePage = require('../../../generated_pages/checkbox_detail_answer_visible/mutually-exclusive.page.js');
+
+
+describe('Given I start a checkbox survey with a write-in option', function() {
```
Comment (LJBabbage) [inline tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js:7]: Shouldn't the focus of the test be detail_answer and visibility not write-in and label.
I would go with something like 
"Given the checkbox detail_answer questionnaire, when a checkbox has a detail_answer with visible set to true then the detail_answer write_in should be shown"

File: test_schemas/en/test_checkbox_detail_answer_visible.json [hunk 3]
```diff
@@ -0,0 +1,195 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Checkbox Visible",
```
[the remark this answers is not in the document]
Comment (petechd) [reply test_schemas/en/test_checkbox_detail_answer_visible.json]: Changed, thanks!

File: tests/app/test_jinja_filters.py [hunk 4]
```diff
@@ -218,6 +219,68 @@ def test_radio_class_visible_attribute():
 
         assert radio.other.open is True
 
+    @staticmethod
+    def test_checkbox_class_visible_attribute():
```
[the remark this answers is not in the document]
Comment (petechd) [reply tests/app/test_jinja_filters.py]: Name is a bit long now, but there are longer in this file😉.

File: tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js (hunk 2 above)
Comment (petechd) [reply tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js]: Added your suggestion, thanks👍

**Pull request merged**